node-src/lib/findChangedDependencies.ts

```diff
@@ -60,29 +60,44 @@ export const findChangedDependencies = async (ctx: Context) => {
 
   ctx.log.debug({ pathPairs }, `Found ${pathPairs.length} manifest/lockfile pairs to check`);
 
-  const tracedPairs = pathPairs.filter(([manifestPath, lockfilePath]) => {
-    if (untraced.some((glob) => matchesFile(glob, manifestPath))) return false;
-    if (untraced.some((glob) => matchesFile(glob, lockfilePath))) return false;
-    return true;
-  });
-  const untracedCount = pathPairs.length - tracedPairs.length;
-  if (untracedCount) {
-    ctx.log.debug(`Skipping ${untracedCount} manifest/lockfile pairs due to --untraced`);
+  // Now filter out any pairs that don't have git changes, or for which the manifest is untraced
+  const filteredPathPairs = pathPairs
```

Comment:
```suggestion
  const filteredPathPairs = metadataPathPairs
```

Reply:
Sorry @skitterm I'm not quite sure what you mean by this comment - are you suggesting we change the name of the first `pathPairs` variable?